dataverse-webapp/src/main/webapp/metrics.xhtml

```diff
@@ -23,38 +23,40 @@
 
             <!-- Published Datasets Chart -->
             <h:form id="publishedDatasetsForm">
-                <h4><h:outputText value="#{bundle['metrics.chart.publishedDatasets.title']}" /></h4>
+                <h4><h:outputText value="#{bundle['metrics.section.datasets.title']}" /></h4>
                 <p:panelGrid>
                     <h:panelGrid columns="3" style="margin-bottom:10px;" cellpadding="25">
                         <p:selectOneRadio id="model" value="#{PublishedDatasetsChart.mode}" style="padding-left: 10px">
                             <p:ajax event="change" update="publishedDatasetsForm" listener="#{PublishedDatasetsChart.changeDatasetMetricsModel}"/>
-                            <f:selectItem itemLabel="#{bundle['metrics.peryear']}" itemValue="YEAR"/>
-                            <f:selectItem itemLabel="#{bundle['metrics.permonth']}" itemValue="MONTH"/>
+                            <f:selectItem itemLabel="#{bundle['metrics.section.datasets.radiobutton.yearly.label']}" itemValue="YEAR"/>
+                            <f:selectItem itemLabel="#{bundle['metrics.section.datasets.radiobutton.yearlyCumulative.label']}" itemValue="YEAR_CUMULATIVE"/>
+                            <f:selectItem itemLabel="#{bundle['metrics.section.datasets.radiobutton.monthly.label']}" itemValue="MONTH"/>
```

Comment:
I think that If we would have cumulative as first option, yearly as second and monthly as third - the whole thing would be more understandable